blog/2015-09-30-HowLuckyAreYourRandomSeeds.adoc

```diff
@@ -0,0 +1,141 @@
+= How lucky are your random seeds?
+:page-interpolate: true
+:awestruct-author: oskopek
+:awestruct-layout: blogPostBase
+:awestruct-tags: [production]
+
+For a long time, it has been speculated that OptaPlanner performs differently when it's random number generator
```

Comment:
"it has been speculated" is pretty negative, it seems like a lot of people have been bad mouthing us (which I am not aware of). Would you changing it to something like "uncertain what the impact of the random seed is".

random number generator != random seed.
I did a blog a long time ago about the impact of the random number generator, to compare JDK random vs mersenne twister etc. See RandomType.java.